### DISCUSSION TITLE
Move from Docker Standalone to Compose

### DISCUSSION THREAD

**Discussion by arn4** [firefly-iii/firefly-iii#7411] [General]
Hello again!

Today I'm here to ask if anybody has ever moved his installation from Docker container + MySQL outside docker to a docker compose one. 
Of course, I'd like to move the installation and all the data. Do you have any suggestion on how to do it properly?

**Comment by JC5** [comment]
Mysql containers come with the "mysqldump" tool. If you start and launch a docker-compose setup, you can use something like `docker exec [container] mysqldump ... ` to import a backup file you made using `mysqldump` on your own machine.

**Comment by arn4** [comment]
Thanks! Successfully migrated
